isis/tests/FunctionalTestsJigsaw.cpp

```diff
@@ -143,4 +145,97 @@ TEST_F(ObservationPair, FunctionalTestJigsawCamSolveAll) {
   // DEC(t3) final
   EXPECT_NEAR(elems.at(56).toDouble(), 0.365717165,          0.00001); 
   
-}
\ No newline at end of file
+}
+
+TEST_F(ObservationPair, FunctionalTestJigsawErrorNoSolve) {
+  // delete to remove old camera for when cam is updated
+  QTemporaryDir prefix;
+  QString outCnetFileName = prefix.path() + "/outTemp.net";
+  QVector<QString> args = {"fromlist="+cubeListFile, "cnet="+cnetPath, "onet="+outCnetFileName, 
+                           "camsolve=None", "spsolve=None"};
+
+  UserInterface options(APP_XML, args);
+  
+  Pvl log; 
+  
+  try {
+    std::cout << "running" << std::endl;
```

Comment:
Similarly, should this be in there, or is this a debug line?

Reply:
nah, debug lines, always forget to trim those